### ISSUE TITLE
LSDIR Testset 2K

### ISSUE THREAD

**Issue by kunncheng** [yongliuy/PatchScaler#1]
Dear author,
Thanks for your nice work.

We noticed that the synthetic test set you used were generated from LSDIR test set consisting of 2000 images. But we found that [LSDIR](https://data.vision.ee.ethz.ch/yawli/index.html) officially didn't make it public.

Could you share your test set here or tell us how to download the test set for LSDIR？

Looking forward to your reply~

**Comment by yongliuy** [comment]
Hi~ 
Thanks for your interest in our work. Since LSDIR hasn't released their official test set, we randomly selected 2,000 images from all of their open source data as LSDIR-TEST, and used the rest of the data as the training set.

**Comment by kunncheng** [comment]
Thank you for your prompt reply.   :)
